client/me/help/help-contact/index.jsx

```diff
@@ -286,7 +295,11 @@ module.exports = React.createClass( {
 		//Autofill the subject field since we will be showing it now that operators have went away.
 		this.autofillSubject();
 
-		this.showAvailabilityNotice( IS_UNAVAILABLE );
+		if ( timedOut ) {
```

Comment:
@dllh Thanks for the steps! I'll run through this too, but having extra 👀 for testing are always welcome.